purchase_work_acceptance_late_fine/models/account_move.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+    _description = "Journal Entries"
+
+    late_wa_id = fields.Many2one(
+        comodel_name="work.acceptance",
+        string="Late WA",
+        domain=lambda self: self._get_fine_amount(),
+    )
+
+    def _get_fine_amount(self):
+        wa = self.env["work.acceptance"].search([("late_fine", ">", 0)])
+        return [("id", "in", wa.ids)]
+
+    @api.onchange("late_wa_id")
+    def _onchange_wa_id(self):
```

Comment:
Naming convention. this can make sure it is not same as what is used before, as the name is unique.
```suggestion
    def _onchange_late_wa_id(self):
```
Note: when write code, please make sure that, our new feature do not affect existing one. For example, we may have to serach the entire system that no method with the same name is used else where.

Don't forget that, Odoo is modular.